crypto/crypto.ts

```diff
@@ -294,11 +294,26 @@ const stdCrypto: StdCrypto = ((x) => x)({
   },
 });
 
-/** FNV (Fowler/Noll/Vo) algorithms names. */
+/**
+ * A FNV (Fowler/Noll/Vo) digest algorithm name supported by std/crypto.
+ *
+ * @deprecated (will be removed in 1.0.0)
+ */
 export type FNVAlgorithms = "FNV32" | "FNV32A" | "FNV64" | "FNV64A";
```

Comment:
The use of this export is minimal. A removal version of v1 instead of sooner is overkill and would be non-impactful to remove sooner. See https://github.com/search?q=%22FNVAlgorithms%22&type=code and notice they're almost entirely forks of std. WDYT, @kt3k?

Reply:
There's no benefit of removing them earlier in my view.

Breaking at 1.0 is more obvious and less confusing. That is better than breaking at random 0.x version in my view